### PR TITLE
53 sakarifeatureprofile removal

### PR DESCRIPTION
**Pull request by Skoivumaki**

### CODE REVIEW

File: frontend-next-migration/src/entities/Clan/index.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ export {
     useCreateClanMutation,
     useDeleteClanMutation,
     useUpdateClanMutation,
+    useDeleteProfileMutation,
```
Comment (leolabdev) [inline frontend-next-migration/src/entities/Clan/index.ts:5]: profile should have its own entity: entities/profile/index.ts

File: frontend-next-migration/src/entities/Clan/model/clanApi.ts [hunk 2]
```diff
@@ -80,6 +82,12 @@ export const clanApi = createApi({
             }),
             invalidatesTags: ['Clan'],
         }),
+        deleteProfile: builder.mutation<void, void>({
```
Comment (leolabdev) [inline frontend-next-migration/src/entities/Clan/model/clanApi.ts:85]: profile should have its own api: entities/profile/model/profileApi

File: frontend-next-migration/src/features/DeleteProfile/model/useDeleteProfile.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { useDeleteProfileMutation } from "@/entities/Clan";
```
Comment (leolabdev) [inline frontend-next-migration/src/features/DeleteProfile/model/useDeleteProfile.ts:1]: entities/Profile

File: frontend-next-migration/src/preparedPages/ProfilePage/ui/ClanAddSubPage/ClanAddSubPage.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (leolabdev) [inline frontend-next-migration/src/preparedPages/ProfilePage/ui/ClanAddSubPage/ClanAddSubPage.tsx:1]: why ClanAddSubPage? should it be some profile page?

File: frontend-next-migration/src/shared/i18n/locales/en/clan.json [hunk 5]
```diff
@@ -12,5 +12,7 @@
   "some_master": "Some Master",
   "head-title": "Clans",
   "head-description": "Learn more about our team and mission.",
-  "head-keywords": "about us, team, mission"
+  "head-keywords": "about us, team, mission",
```
Comment (leolabdev) [inline frontend-next-migration/src/shared/i18n/locales/en/clan.json:15]: [lng]/profile.json

File: frontend-next-migration/src/shared/i18n/locales/fi/clan.json [hunk 6]
```diff
@@ -1,21 +1,17 @@
 {
```
Comment (leolabdev) [inline frontend-next-migration/src/shared/i18n/locales/fi/clan.json:1]: [lng]/profile.json

**Review by leolabdev — CHANGES_REQUESTED** [review]

**Comment by Skoivumaki** [comment]
Was requested to do it quick so I did it QUICK. Renamed fiels disabled some code related to profile deletion API, will be useful when profile overview is done properly. Maybe this is the next thing.

**Review by leolabdev — APPROVED** [review]

**Pull request merged**